tests/go.mod

```diff
@@ -0,0 +1,156 @@
+module sigs.k8s.io/aws-fsx-openzfs-csi-driver/tests
```

Comment:
In next revision going to remove this go.mod file. We shouldn't have a separate module for testing.